src/js/components/account/visualizations/sankey/SankeyVisualizationHorizontal.jsx

```diff
@@ -88,8 +88,10 @@ export default class SankeyVisualizationHorizontal extends React.Component {
         this.generateChart(this.props);
     }
 
-    componentWillReceiveProps(nextProps) {
-        this.generateChart(nextProps);
+    componentDidUpdate(prevProps) {
+        if (this.props !== prevProps) {
```

Comment:
the deep equality comparison operator `===` or `!==` doesn't apply to objects. 
![image](https://user-images.githubusercontent.com/12897813/94450694-9063d900-017b-11eb-861f-ee32bc5ecc98.png)

We should. use lodash's `isEqual` instead,